code/src/index.css

```diff
@@ -1,13 +1,189 @@
     monospace;
 }
+
+.outer-wrapper {
+  padding: 120px 10px;
+  background-color: #0e0e0e;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.inner-wrapper {
+  background-color: #181818;
+  padding: 15px;
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1200px;
+}
+
+.albums-wrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+}
+
+.album-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start; /* Makes it so the link is not the full width */
+  padding: 20px 10px;
+}
+
+.title {
+  font-size: 18px;
+  color: white;
+  width: 100%;
+  margin: 0;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(44, 44, 44);
+}
+
+.album-name {
+  font-family: Helvetica, sans-serif;
+  color: white;
+  text-decoration: none;
+  font-size: 14px;
+  margin: 2px 0;
+}
+.album-name:hover {
+  text-decoration: underline;
+}
+
+.artist-name {
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+
+  margin: 0;
+  color: #a0a0a0;
+  text-decoration: none;
+}
+
+/* Add a comma before every child that is not the first nor last. */
+.artist-name-container > :not(:last-child):not(:first-child):before {
+  content: ', ';
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+/* Add a & before every child that is the last but not also first. */
+.artist-name-container > :not(:first-child):last-child:before {
+  content: ' & ';
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+.artist-name:hover {
+  color: rgb(216, 216, 216);
+  text-decoration: underline;
+}
+
+.album-cover-container {
+  position: relative;
+  opacity: 1;
+  transition: opacity .5s;
+}
+
+.icons-container {
+  top: 50%;
+  left: 50%;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  column-gap: 20px;
+  opacity: 0;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  text-align: center;
+  transition: opacity .3s;
+}
+
+.album-cover {
+  opacity: 1;
+  transition: .3s;
+  width: 250px; 
+}
+
+.album-cover-container:hover .album-cover {
+  opacity: 0.3;
+  transition: .3s;
+}
+
+.album-cover-container:hover .icons-container {
+  opacity: 1;
+  transition: .3s;
+}
+
+.icon {
+  width: 30px;
+  filter: invert(0.8);
+  transition: .3s;
+}
+
+.play-icon {
+  width: 70px;
+}
+
+.play-icon:hover {
+  filter: invert(1);
+  transform: scale(1.1);
+  transition: 0.3s;
+}
+
+.heart-icon:hover, .dots-icon:hover {
+  filter: invert(1);
+  transition: 0.2s ease-in-out;
+} 
+
+/* Media queries */
+
+  /* DESKTOP */
+  @media (max-width: 1400px) {
+    .album-cover {
+      width: 200px;
+    }
+  }
+
+  /* TABLET */
+  @media (max-width: 1100px) {
+    .albums-wrapper {
+      grid-template-columns: 1fr 1fr;
+    }
+    .album-cover {
+      width: 300px;
+    }
+  } 
+
+  @media (max-width: 830px) {
+    .album-cover {
+      width: 250px;
+    }
+  } 
+
+  /* MOBILE */
+  @media (max-width: 700px) {
```

Comment:
Why not use standard measurements?

Reply:
Hi! Good question :)
We decided on using extra breakingpoints because of the width of the images. For us, the container becomes too small to nicely fit 2 albums next to eachother.